### PR TITLE
Hello world

### PR DESCRIPTION
**Pull request by Carlosmatilla**
Hello world babababababababab

### CODE REVIEW

File: staff/carlos-matilla/hello-world/index.html [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Hello, World!</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <h1>Hello, World!</h1>
```
Comment (Carlosmatilla) [inline staff/carlos-matilla/hello-world/index.html]: bla bla bla

File: staff/carlos-matilla/hello-world/style.css [hunk 2]
```diff
@@ -0,0 +1,7 @@
+body{
+    background-color: yellow;
+}
+
+h1{
+    cursor: pointer;
```
Comment (Carlosmatilla) [inline staff/carlos-matilla/hello-world/style.css]: bla bla bla

File: staff/carlos-matilla/hello-world/index.html (hunk 1 above)
Comment (serbittan) [reply staff/carlos-matilla/hello-world/index.html]: todo blah blah

File: staff/carlos-matilla/hello-world/index.html [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+    <p>me se hantoja un vocadiyo de vacon haumado</p>
```
Comment (serbittan) [inline staff/carlos-matilla/hello-world/index.html]: revisa la ortografia. Aqui te has colao

**Pull request merged**